src/components/DataRoom/CryptoPunks/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function getColor() {
+  const chance = Math.random()
+
+  if (chance <= 0.24) return '#12FF80' // Green (14% CHANCE)
+  if (chance <= 0.17) return '#12A154' // Dark Green (3% CHANCE)
+  if (chance <= 0.2) return '#124228' // Darker Green (3% CHANCE)
+
+  return 'currentColor'
```

Comment:
Why do we include chance in this? It would be a lot simpler to just have an array and pick a random index.

Reply:
This effect including different probabilities was tuned with @liliiaorlenko . Does the array implementation make sense nevertheless?

Reply:
If it's a necessity then I'd say it's fine as is but the chance values seemed quite arbitrary. Using a random index will never mean you'll have to fallback to `currentColor`. This is what I was alluding to:

```suggestion
  const colors = ['#12FF80', '#12A154', '#124228']
  return colors[Math.floor((Math.random() * colors.length))]
```

Reply:
But sometimes you don't want to color the SVG. I understand the chance values seem random but I will trust @liliiaorlenko and @Malayvasa  came to these values together. Can you confirm @Malayvasa ?